galgebra/mv.py

```diff
@@ -2180,13 +2178,8 @@ def __eq__(self, dop):
             if self.Ga != dop.Ga:
                 return False
 
-            self = Sdop.consolidate_coefs(self)
-            dop = Sdop.consolidate_coefs(dop)
-            if len(self.terms) != len(dop.terms):
-                return False
-            if set(self.terms) != set(dop.terms):
-                return False
-            return True
+            diff = self - dop
+            return all(coef == S(0) for coef, _ in diff.terms)
```

Comment:
Would be neat if this could be `len(diff.terms) == 0`, but that will need changes to `Dop.Add`, which I don't want to make yet.